app/assets/javascripts/initializers/runtime.js

```diff
@@ -20,16 +20,13 @@ class Runtime {
   /**
    * This function returns a string that represents the current Medium where
    * the app is currently running. The currently supported mediums are Browser,
-   * ForemWebView and PWA.
+   * and ForemWebView.
    *
    * @returns {String} One of the supported Mediums or 'Unsupported'
    */
   static currentMedium() {
```

Comment:
(suggestion / not blocking) now there are only two options we could simplify this if we want. Something like

```js
return /ForemWebView/i.test(navigator.userAgent) ? 'ForemWebView' : 'Browser';
```

Reply:
done!